modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/ScalaPlayFrameworkServerCodegen.java

```diff
@@ -1,3 +1,19 @@
+/*
+ * Copyright 2018 OpenAPI-Generator Contributors (https://openapi-generator.tech)
```

Comment:
Should the file name be updated to remove `Framework` as well?

Reply:
I thought about that too but there are some other generators not following a similar naming convention. (looks like we never enforce it)

I may file another PR to clean those up before the 4.0.0 release.